WooCommerce/Classes/ViewRelated/Fancy Alerts/FancyAlertViewController+UnifiedLogin.swift

```diff
@@ -21,9 +22,14 @@ extension FancyAlertViewController {
         return controller
     }
 
-    static func makeNeedHelpFindingEmailAlertController() -> FancyAlertViewController {
+    static func makeNeedHelpFindingEmailAlertController(screen: CustomHelpCenterContent.Screen? = nil) -> FancyAlertViewController {
         let dismissButton = makeDismissButtonConfig()
-        let moreHelpButton = makeNeedMoreHelpButton()
+        var customHelpCenterContent: CustomHelpCenterContent?
+        if let screen = screen {
+            customHelpCenterContent = CustomHelpCenterContent(screen: screen,
+                                                              flow: AuthenticatorAnalyticsTracker.shared.state.lastFlow)
+        }
```

Comment:
nit: maybe this can be simplified a bit

```suggestion
        let customHelpCenterContent: CustomHelpCenterContent? = screen
            .map { CustomHelpCenterContent(screen: $0, flow: AuthenticatorAnalyticsTracker.shared.state.lastFlow) }
```

Reply:
Nice idea. Done in https://github.com/woocommerce/woocommerce-ios/pull/7769/commits/de51a0a51aa8fc9a231e1adee3ab7e2462e37a52